msteams-platform/webhooks-and-connectors/how-to/connectors-using.md

```diff
@@ -61,7 +61,14 @@ You can also use this JSON to create cards containing rich inputs, such as text
             "@type": "HttpPOST",
             "name": "Save",
             "target": "http://..."
-        }]
+        },
+        {
+      "@type": "OpenUri",
+      "name": "Learn More",
+      "targets": [
+        { "os": "default", "uri": "https://docs.microsoft.com/outlook/actionable-messages" }
+      ]
+    }]
```

Comment:
Check and correct the alignment of the braces and content with the previous sections braces and content.

Reply:
Corrected